visualization/package.json

```diff
@@ -76,7 +76,7 @@
 		"lint": "tslint -c \"tslint.json\" \"app/**/*.ts\" --project tsconfig.json",
 		"lint:fix": "tslint -c \"tslint.json\" \"app/**/*.ts\" --fix --project tsconfig.json",
 		"format": "prettier --write \"./**/*\"",
-		"format:quick": "pretty-quick --pattern \"./**/*\" --staged"
+		"format:quick": "pretty-quick --staged"
```

Comment:
What is the difference here? Did it format all files?

Reply:
For some reason the pattern was ignoring some files. Seems like the pattern can't use something that just consists of wildcards. In this case, leave the pattern empty.